dashboard/dashboard.cfm

```diff
@@ -181,11 +181,25 @@
 					<cfloop from="1" to="#arrayLen(application._taffy.uriMatchOrder)#" index="local.resource">
 						<cfset local.currentResource = application._taffy.endpoints[application._taffy.uriMatchOrder[local.resource]] />
 						<cfset local.resourceHTTPID = rereplace(local.currentResource.beanName & "_" & hash(local.currentResource.srcURI), "[^0-9a-zA-Z_]", "_", "all") />
+						<cfset local.md = getMetaData(application._taffy.factory.getBean(local.currentResource.beanName)) />
+						<cfif structKeyExists(local.md, "taffy_dashboard_hide") OR structKeyExists(local.md, "taffy:dashboard:hide")>
+							<cfscript>continue;</cfscript>
```

Comment:
Is this a cf8 compat thing? I see cf9 added `<cfcontinue />` but it was available in 8 first?

Reply:
Correct. As of CF8, this was literally the only thing you could _only_ do in cfscript.